### ISSUE TITLE
"Ignore Users" boolean for raycasts.

### ISSUE THREAD

**Issue by VirgilCore** [Yellow-Dog-Man/Resonite-Issues#3415]
### Is your feature request related to a problem? Please describe.

There are certain things I want to recreate, that require raycasts that do not hit users. For instance projected tractor beams or bridges. (Like in Portal 2)

### Describe the solution you'd like

A simple boolean input on raycast related nodes (and sweepcasts whenever those are implimented) similar to the input that exists already to *only* hit users, except for the inverse, to *never* hit users.

### Describe alternatives you've considered

Having dozens of seperate raycasts and using the maximum hit distance out of all of them

### Additional Context

I was told there used to be a component in frooxengine for this many years ago, but I don't actually know if that was true.

### Requesters

anquietas_sys

**Comment by Nytra** [comment]
Closest thing I think is the Raycaster component which has an IgnoreHierarchy option, you can put an entire user slot in there, but this would only really work if you put all users under one slot in the first place

**Comment by VirgilCore** [comment]
Yeah, like you said this doesnt really help for this specific situation, especially if two people are in the path of the beam.

**Comment by Nytra** [comment]
Not sure if you saw the edit

> Edit: actually you can get a FilterUsers sync delegate from the MeterTool which can be placed into the Filter field

**Comment by VirgilCore** [comment]
I think this was the thing i was told about years ago but when I tried it back then it didnt exist any more. Couls someone verify if this is still the case?

**Comment by Nytra** [comment]
Works for me

**Comment by VirgilCore** [comment]
Is there documentation on the wiki on how that's used? Its existance still doesnt really solve the problem of ease of use since its a very obscure component.

**Comment by Nytra** [comment]
Not sure, I found it with a decompiler

**Comment by VirgilCore** [comment]
To return to my original request, I feel like there should be an easy boolean to exclude users from a raycast just like how we have a boolean to only hit users.

**Comment by Redd56** [comment]
this has once again come up and i desperately need it for simplified culling systems
